packages/web-client/app/routes/card-pay/payments.ts

```diff
@@ -23,7 +22,7 @@ const MERCHANTS_PANEL = {
         'Confirm transactions with Touch ID or Face ID in the Card Wallet mobile app',
       ],
       cta: 'Create Payment Profile',
-      isCtaDisabled: !config.features.createMerchant,
+      isCtaDisabled: false,
```

Comment:
Maybe just remove? But maybe also who cares 😆

Reply:
I was wondering about removing that too, but decided against it since this pattern of introducing things and then disabling them seems to be a thing.